.gitignore

```diff
@@ -57,6 +57,10 @@ Makefile.in
 /src/view_file/.deps
 /src/view_file/.dirstamp
 
+# /subprojects
+/subprojects/packagecache
```

Comment:
What is it for?

Reply:
When meson downloads gtest based on the wrap file, it will create those directories.  Those files should not be checked in:
```
$ls subprojects/*
subprojects/gtest.wrap

subprojects/googletest-release-1.10.0:
appveyor.yml  CMakeLists.txt   googletest/   LICENSE.build   README.md
BUILD.bazel   CONTRIBUTING.md  library.json  meson.build     WORKSPACE
ci/           googlemock/      LICENSE       platformio.ini

subprojects/packagecache:
gtest-1.10.0-1-wrap.zip  gtest-1.10.0.zip
```